runtime/doc/mbyte.txt

```diff
@@ -494,7 +494,7 @@ Char UTF-8 cp1255  hebrew  hebrewp  name ~
 ן    0x5df  0xef     i	      N     nun sofit
 נ    0x5e0  0xf0     b	      n     nun
 ס    0x5e1  0xf1     x	      s     samech
-ע    0x5e2  0xf2     g	      u     `ayin
+ע    0x5e2  0xf2     g	      u     ayin
```

Comment:
do quotes work for this case? seems like the backtick is an important detail

Reply:
(I googled, and Wikipedia has these letters without diacritics (as opposed to the Arabic ones of the same or similar name); I removed the forward tick for symmetry.)